### ISSUE TITLE
how to use coc-html for jquery

### ISSUE THREAD

**Issue by laoshaw** [neoclide/coc-html#17]
I notice jquery ts.d is included, I CocInstall coc-install and put the below to coc-settings:

```
    "html.format.enable": true,
    "html.validate.scripts": true,
    "html.validate.styles": true,
    "html.autoClosingTags": true,
```

However when  I edit a html or js file I don't see jquery autocomplete, e.g. `$("li").` will have nothing shown on dropdown list after pressing TAB. I did not see any jQuery autocomplete(i.e. jQuery keywords) either.

**Comment by chemzqm** [comment]
It has jquery support.

**Comment by laoshaw** [comment]
what are the steps to make jquery work with coc-html? I did the steps above but there is no jquery autocomplete whatsoever. 

does it have to be a *.js file, or script inside *.html will work too?

all my other languages are working, including javascript actually(by tsserver), but nothing happened to jquery yet. do I need install html language server somewhere or is it installed by `CocInstall coc-html` automatically? can you give a working coc-settings.json for coc-html?

Thanks,

**Comment by chemzqm** [comment]
It should just work, but looks like it's broken for some reason.

**Comment by phaalonso** [comment]
I have the same problem ;(

**Comment by laoshaw** [comment]
if I have jquery installed via npm I can get jquery working(without coc-html actually) in js files, but for javascript embedded in html file there is no way to get autocompletion working.

**Comment by yesyestoday2020** [comment]
You want this kind of effect

<img width="1440" alt="截屏2020-09-13 04 48 10" src="https://user-images.githubusercontent.com/35359288/93004721-5b9d3280-f57c-11ea-883c-f3e585678901.png">

my use vscode snippet  -- jquerysnippets.

**Comment by chemzqm** [comment]
Fixed on coc-html@1.3.0
